src/routes/FavoritesScreen.tsx

```diff
@@ -33,14 +34,16 @@ const FavoritesScreen = ({ navigation }: FavoritesScreenProps): ReactElement =>
 
   return (
     <ServerResponseHandler error={error} loading={false} refresh={refresh}>
-      {data && (
-        <VocabularyList
-          title={labels.favorites}
-          documents={data}
-          onFavoritesChanged={refresh}
-          onItemPress={onItemPress}
-        />
-      )}
+      <RouteWrapper>
```

Comment:
I think RouteWrapper should be outside and ServerResponseHandler inside, because Loading and ErrorStuff should be inside RouteWrapper. Same for other routes.